srcpkgs/ktorrent/template

```diff
@@ -1,20 +1,26 @@
 # Template file for 'ktorrent'
 pkgname=ktorrent
-version=5.2.0
+version=20.12.1
 revision=1
 build_style=cmake
+configure_args="-DWITH_SYSTEM_GEOIP=ON"
 hostmakedepends="extra-cmake-modules qt5-host-tools qt5-qmake gettext"
 makedepends="qt5-devel qt5-script-devel ki18n-devel kio-devel kxmlgui-devel
  knotifications-devel ksolid-devel kconfig-devel kcoreaddons-devel kdoctools-devel
  kross-devel kcmutils-devel sonnet-devel kdbusaddons-devel kcrash-devel
  karchive-devel knotifyconfig-devel libktorrent-devel kplotting-devel
- gmp-devel taglib-devel kdnssd-devel kdewebkit-devel"
+ gmp-devel taglib-devel kdnssd-devel kdewebkit-devel geoip-devel syndication-devel"
+depends="geoip-data"
```

Comment:
We are dropping `geoip-data` package for the same reason: can't download it anymore.

Reply:
> We are dropping `geoip-data` package for the same reason: can't download it anymore.

It is pretty easy to create the data in the same format - I can try to create a package if you think it will be usefull. Without it the user gets an error message **every time** ktorrent starts. That is why I did the relevant changes.

Reply:
If there is good source to create it form then maybe, but see #27445. All other dependants are leaving it too.
Also please talk to upstream on their plan.

Reply:
There is a good source to get dbs in legacy format : https://mailfud.org/geoip-legacy/
It is what archlinux uses:

- https://github.com/archlinux/svntogit-packages/tree/packages/geoip-database/trunk
- https://github.com/archlinux/svntogit-packages/tree/packages/geoip-database/trunk

I whould rather keep the change... I cannot find a way to disable the feature altogether and having an error earch time the program starts is problematic.

Reply:
I removed geoip-data from depends since it is removes from the repo and added an INSTALL.msg file for obtaining the data with geoipupdate. The programms works :)